views/main/play-settings.ejs

```diff
@@ -18,102 +18,50 @@
     >
       <div class="setting">
         <h5 class="setting-title">Difficulty</h5>
-        <fieldset class="setting-fieldset" name="difficulty">
-          <input
-            class="field-input"
-            type="radio"
-            name="difficulty"
-            value="1"
-            id="difficulty-1"
-            onchange="chooseDifficulty(this)"
-          />
-          <label for="difficulty-1">Very Easy</label>
-          <input
-            class="field-input"
-            type="radio"
-            name="difficulty"
-            value="2"
-            id="difficulty-2"
-            onchange="chooseDifficulty(this)"
-          />
-          <label for="difficulty-2">Easy</label>
-          <input
-            class="field-input"
-            type="radio"
-            name="difficulty"
-            value="3"
-            id="difficulty-3"
-            onchange="chooseDifficulty(this)"
-          />
-          <label for="difficulty-3">Medium</label>
-          <input
-            class="field-input"
-            type="radio"
-            name="difficulty"
-            value="4"
-            id="difficulty-4"
-            onchange="chooseDifficulty(this)"
-          />
-          <label for="difficulty-4">Hard</label>
-          <input
-            class="field-input"
-            type="radio"
-            name="difficulty"
-            value="5"
-            id="difficulty-5"
-            onchange="chooseDifficulty(this)"
-          />
-          <label for="difficulty-5">Very Hard</label>
+        <fieldset
+          class="setting-fieldset"
```

Comment:
At the moment, options don't wrap very well: 
![image](https://github.com/DmitrijSidorkin/pixel-thing/assets/9388169/81da17ed-13e7-4fa8-a1cc-93cabe2d1c59)